.github/scripts/rust/Makefile.toml

```diff
@@ -12,6 +12,10 @@ CARGO_RUN_FLAGS = ""
 CARGO_FORMAT_FLAGS = ""
 CARGO_FORMAT_HACK_FLAGS = "--workspace"
 CARGO_CLIPPY_FLAGS = "--no-deps --tests"
+# This needs to be here, as error-stack uses it, and _BASE **needs** to be defined before
+# the variable using it has been defined. This is done per suggestion of
+# https://github.com/sagiegurari/cargo-make/issues/685
+CARGO_CLIPPY_HACK_FLAGS_BASE = ""
```

Comment:
This is annoying but probably the easiest way for us 🙁

Reply:
We will be able to remove this once https://github.com/sagiegurari/cargo-make/pull/688 is merged.

Reply:
Can't wait for it 😆